core/src/proxy/relayer_stage.rs

```diff
@@ -213,25 +242,15 @@ impl RelayerStage {
 
         datapoint_info!(
             "relayer_stage-tokens_generated",
-            (
-                "url",
-                relayer_config.auth_service_endpoint.uri().to_string(),
-                String
-            ),
+            ("url", local_config.relayer_url, String),
```

Comment:
```suggestion
            ("relayer_url", local_config.relayer_url, String),
```

a bit vague when just looking at influx

Reply:
gonna leave this to avoid breaking anything that reads the influx tag